### PR TITLE
Js

### PR DESCRIPTION
**Pull request by tyndria**

### CODE REVIEW

File: JavaScript.js [hunk 1]
```diff
@@ -0,0 +1,233 @@
+var photoPosts = [
```
Comment (tyndria) [inline JavaScript.js]: JavaScript.js - name is not so good, you could name this file 'main.js', for example

File: JavaScript.js [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+            }
+        }
+
+        return -1;
+    }
+
+    function getPhotoPost(value) { // 2й вариант
+        a = photoPosts.findIndex(item => item.id === value);
+        if(a !== -1 )
+        {
```
Comment (tyndria) [inline JavaScript.js]: Please, be accurate with spaces, tabs, parentheses. I meat, you if-else statement is a little bit ugly:
if () {
} else {
}
 ---- this is much better
And decide, what you want to return in a case of 'negative' result: -1 or false or something else

File: JavaScript.js [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+
+                return photoPosts[i];
+            }
+        }
+
+        return -1;
+    }
+
+    function getPhotoPost(value) { // 2й вариант
+        a = photoPosts.findIndex(item => item.id === value);
```
Comment (tyndria) [inline JavaScript.js]: Read about difference between using keyword 'var' and absent of it when you define variable, please

File: JavaScript.js [hunk 4]
```diff
@@ -0,0 +1,233 @@
...
+        if((typeof photoPosts.author !== "string") || photoPosts.author == '') {
+            return -1;
+        }
+        if((typeof photoPosts.photoLink !== "string") || photoPosts.photoLink == '') {
+            return -1;
+        }
+        return true;
+    }
+
+    function addPhotoPost(photoPosts1) {
```
Comment (tyndria) [inline JavaScript.js]: perhaps, it's better to name argument 'photoPost', isn't it?

**Comment by tyndria** [comment]
there are no 'tests' to check your code. I mean, you don't call your functions with arguments in you .js file

File: UI/main.js [hunk 5]
```diff
@@ -0,0 +1,391 @@
...
+        function checkHashtags(photoPosts) {
+            return photoPosts.hashtags.every(function (item) {
+                return item.charAt(0) === "#";
+            });
+        }
+        return true;
+    }
+
+
+    let addPhotoPost = function(PhotoPosts) {
```
Comment (tyndria) [inline UI/main.js]: why does you argument has capital letter? and why it's ``PhotoPosts``? you could add multiple photo posts at once?

**Comment by tyndria** [comment]
I could edit field likes, tags... Please, check this. This field should be unchangeable

File: UI/main.js (hunk 5 above)
Comment (GrishinN) [reply UI/main.js]: исправил , насчет лайков согласен мой косяк , но я предполагал что хэштеги можно изменять

**Pull request merged**